### ISSUE TITLE
VMOptions javaagent Parameter directory error

### ISSUE THREAD

**Issue by HyWell** [tofi86/universalJavaApplicationStub#84]
I'm submitting a…

- [x] bug report
- [ ] feature request
- [ ] other

**Short description of the issue/suggestion:**
VMOptions javaagent Parameter directory error

**Steps to reproduce the issue/enhancement:**

1. edit plist.info
2. set vmoptions javaagent
`
<key>VMOptions</key>
      <array>
          <string>-javaagent:Contents/Resources/Loader.jar</string>
      </array>
`
3. Contents/MacOS/universalJavaApplicationStub

**What is the expected behavior?**
Loader.jar can be found

**What is the current behavior?**
Error opening zip file or JAR manifest missing

**Do you have screenshots, GIFs, demos or samples which demonstrate the problem or enhancement?**
![image](https://user-images.githubusercontent.com/21263451/80609509-bd944a00-8a6a-11ea-9a09-23fa849d2396.png)

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

- universalJavaApplicationStub version: 3.0.6
- Mac OS version: 10.15.4 
- Java version(s): openJDK11

**Other information** (e.g. related issues, suggestions how to fix, links for us to have context)
use getfullpath is work, for example `javaagent:/Users/hywell/Loader.jar`

**Comment by tofi86** [comment]
Thanks for the report. There are a couple of other keys (`WorkingDirectory`, `ClassPath`, `Arguments`, `Properties`) where we replace occurances of `$APP_PACKAGE`, `$JAVAROOT` and `$HOME`, but not (yet) on the `VMOptions` key.

But I could add this as an enhancement so you would be able to use it as follows:

```xml
<key>VMOptions</key>
<array>
  <string>-javaagent:"$APP_PACKAGE/Contents/Resources/Loader.jar"</string>
</array>
```

Not sure yet when to find time for this and for testing... Please give me a few days...

**Comment by tofi86** [comment]
Fixed with cbee3f056d398e0b1f20131d67857f8c9acb9d48

You can use it with my example from above.

Sorry for the huge delay! It has been a tough year for all of us... :-(

**Comment by tofi86** [comment]
Version 3.1.0 has officially been released 🎉

Details: https://github.com/tofi86/universalJavaApplicationStub/releases/tag/v3.1.0

Note: The release now also contains compiled binaries which seem to be necessary in macOS 11.0 "Big Sur" and later...
